build_all/mbed5/build.sh

```diff
@@ -1,20 +1,18 @@
 # Copyright (c) Microsoft. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
+git clone -b rajeev/latest_mbed https://github.com/massand/mxchip_az3166_firmware.git mxchip
+# rsync -avz --existing ./ mxchip/mbed-iot-devkit-sdk/cores/arduino/azure-iot-sdk-c/
 
 sudo npm install -g iotz
 sudo iotz update
 
-sudo rm -R -f mxchip
```

Comment:
is there a reason to remove these lines? looks like the `rm` is used to clean up the folder just in case.

Reply:
It probably could remain in with no issues.